pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/fredeil/email-validator.dart
 description: A simple (but correct) dart class for validating email addresses
   
 environment:
-  sdk: '>=1.24.0 <3.0.0'
+  sdk: '2.12.0-33.0.dev <3.0.0'
```

Comment:
I think, you should use `sdk: ">=2.12.0-0 <3.0.0"` instead hard-coding version.

Reply:
This can just be `2.12.0-0`.